daemon/loadbalancer.go

```diff
@@ -184,7 +98,7 @@ func NewDeleteServiceIDHandler(d *Daemon) DeleteServiceIDHandler {
 func (h *deleteServiceID) Handle(params DeleteServiceIDParams) middleware.Responder {
 	log.WithField(logfields.Params, logfields.Repr(params)).Debug("DELETE /service/{id} request")
 
-	found, err := h.d.svcDeleteByID(loadbalancer.ServiceID(params.ID))
+	found, err := h.d.svc.DeleteServiceByID(loadbalancer.ServiceID(params.ID))
```

Comment:
Instead of having `deleteServiceID` have a `*Daemon`, it should just have a `*svc.Service` member. That way we can just do `h.svc.DeleteServiceByID` - we could use this to make it easier to test our API handler functions in the future, for instance.

Reply:
Neat idea! Going to change in all `daemon/loadbalancer.go` handler.